schema/measurement_methods.mcf

```diff
@@ -36,3 +36,21 @@ name: "ConstantMaturityRate"
 typeOf: dcs:BondYieldMeasurementEnum
 description: "Indicates that the yield data are extrapolated against a yield curve, which relates the yield on a security to its time to maturity, to estimate the yield of the bond if it had exactly some maturity"
 descriptionUrl: "https://www.investopedia.com/terms/c/cmtindex.asp"
+
+Node: dcid:ChainingEnum
+name: "ChainingEnum"
+typeOf: schema:Class
+subClassOf: schema:MeasurementMethodEnum
+description: "An enumeration of chained and unchained price index measurement methods"
```

Comment:
Super nit: period at the end, ditto below

Reply:
added